discussion.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Discussion
+---
+
+Frequently Asked Questions
```

Comment:
I don't know if we have a written guideline, but `##` is definitely what I see elsewhere in our lessons.